packages/babel-core/test/targets.js

```diff
@@ -91,11 +91,24 @@ describe("browserslist", () => {
     ).toEqual({ chrome: "80.0.0" });
   });
 
-  it("loads .browserslistrc relative to the input file", () => {
+  it("loads .browserslistrc relative to the root", () => {
     expect(
       loadOptions({
         cwd: join(cwd, "fixtures", "targets"),
-        filename: "./nested/test.js",
+        filename: "./node_modules/dep/test.js",
+      }).targets,
+    ).toEqual({ chrome: "80.0.0" });
+  });
+
+  // TODO: browserslistConfig is currently resolved starting from the root
+  // rather than from the config file.
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip("loads nested .browserslistrc files if explicitly specified", () => {
```

Comment:
This is a separate bug, I found it while attempting to write a test. I'll fix it in a follow-up PR.

Reply:
#13031